src/data/aws_compliance_actions.py

```diff
@@ -53,9 +55,8 @@ def apply(self) -> ComplianceActionReport:
     def _apply(self) -> Optional[Dict[str, Any]]:
         """"""
 
-    @abstractmethod
     def plan(self) -> ComplianceActionReport:
-        """"""
+        return ComplianceActionReport(description=self.description)
```

Comment:
Just checking this is correct?  Seeing as no return in the apply bit?

Reply:
Yes, this is because `apply` is an abstract method and `plan` isn't.